MyBlog.API/Controllers/UserController.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using MyBlog.Infrastructure.Entities;
+using MyBlog.Service.Areas.Users;
+
+namespace MyBlog.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class UserTestController : ControllerBase
+{
+    private readonly ILogger<UserTestController> _logger;
+    private readonly IUserService _userService;
+    
+    public UserTestController(ILogger<UserTestController> logger, IUserService userService)
+    {
+        _logger = logger;
+        _userService = userService;
+    }
+
+    /// <summary>
+    /// Get list of Users
+    /// </summary>
+    /// <returns>Returns User list</returns>
+    /// <response code="200">Success</response>
+    [HttpGet]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    public async Task<IActionResult> GetListAsync()
+    {
+        var users = await _userService.GetListAsync();
+        
+        return Ok(users);
+    }
+
+    /// <summary>
+    /// Get User by id
+    /// </summary>
+    /// <param name="id">User id</param>
+    /// <returns>Returns User</returns>
+    /// <response code="200">Success</response>
+    /// <response code="404">UserNotFound</response>
+    [HttpGet( "{id:int}")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    public async Task<IActionResult> GetByIdAsync(int id)
```

Comment:
if you write in the tag of your method(httpget, httpupdate or someone else route parameter (in your case it`s id:int), so in the method parameters you should specify, that you take id from route: GetByIdAsync([FromRoute] int id))

Apply this rule for all methods in your controller